### PR TITLE
Add stub ensemble backend

### PR DESCRIPTION
**Pull request by GuanLuo**

### CODE REVIEW

File: src/servables/ensemble/ensemble_bundle.proto [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+syntax = "proto3";
+
+package nvidia.inferenceserver;
+
+// Config proto for EnsembleBundleSourceAdapter.
+message EnsembleBundleSourceAdapterConfig
+{
+  // An variable to store pointer to inference server, which will be used
+  // in ensemble scheduler
+  uint64 inference_server = 1;
```
Comment (deadeyegoodwin) [inline src/servables/ensemble/ensemble_bundle.proto]: Is uint64 the supported way to pass a pointer via protobuf. Generally that is bad practice...

Comment (GuanLuo) [reply src/servables/ensemble/ensemble_bundle.proto]: No, protobuf doesn't provide any instruction on storing pointers, maybe using "bytes" if we still want to passing the pointer via protobuf.

Another way is adding a new setter function in InferenceBackend and passing the pointer before calling `Run()`

File: src/core/server.cc [hunk 2]
```diff
@@ -806,6 +806,7 @@ InferenceServer::HandleInfer(
     OnCompleteInferRPC();
   };
 
+  backend->Backend()->SetInferenceServer(this);
```
Comment (deadeyegoodwin) [inline src/core/server.cc]: You should put a comment here: "Need to set 'this' in each backend even though it is redundant after the first time. Once we remove TFS dependency we can construct each backend in a way that makes it directly aware of the inference server"

**Comment by deadeyegoodwin** [comment]
@GuanLuo You need to manually rebase this to master before it can be merged.

**Pull request merged**